### PR TITLE
POSHI-588  Run test case under cursor doesn't work with case with number in name

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [POSHI-588](https://issues.liferay.com/browse/POSHI-588).
This is an update for [POSHI-589](https://liferay.atlassian.net/browse/POSHI-589).

cc @kenjiheigel

### CODE REVIEW

**Comment by drewbrokke** [comment]
ci:test:sf

**Comment by drewbrokke** [comment]
@kenjiheigel if this looks good to you then we can forward.

File: modules/test/poshi/poshi-vscode/src/lib/tokens.ts [hunk 1]
```diff
@@ -31,12 +31,12 @@ const tokenPatternMap = {
 	 * matches "PathFile"
 	 * matches "PathFile|Name.LOCATOR_NAME"
 	 */
-	pathFileName: /"([A-Z][A-Za-z]+)/g,
+	pathFileName: /"([A-Z][A-Za-z]+)#/g,
```
Comment (kenjiheigel) [inline modules/test/poshi/poshi-vscode/src/lib/tokens.ts:34]: Should also be `\w-`.

File: modules/test/poshi/poshi-vscode/src/lib/tokens.ts [hunk 2]
```diff
@@ -31,12 +31,12 @@ const tokenPatternMap = {
...
 
 	/**
 	 * matches PathFileName.LOCATOR_N|AME
 	 */
-	pathLocator: /"([A-Z][A-Za-z]+)#([A-Z][A-Z_-]+)"/g,
+	pathLocator: /"([A-Z][A-Za-z]+)#([A-Z][A-Z0-9_-]+)"/g,
```
Comment (kenjiheigel) [inline modules/test/poshi/poshi-vscode/src/lib/tokens.ts:39]: `"([\w-]+)#([A-Z][A-Z0-9_-]+)"`

File: modules/test/poshi/poshi-vscode/src/lib/tokens.ts [hunk 3]
```diff
@@ -55,8 +55,8 @@ const tokenPatternMap = {
 	/**
 	 * matches ClassName.method|Name
 	 */
-	methodInvocation: /[^\w.]([A-Z][A-Za-z]+)\.([A-Za-z_][A-Za-z]+)/g,
-	methodDefinition: /(?:macro|function) ([A-Za-z_][A-Za-z]+) \{/g,
+	methodInvocation: /[^\w.]([A-Z][A-Za-z]+)\.([\w-]+)/g,
```
Comment (kenjiheigel) [inline modules/test/poshi/poshi-vscode/src/lib/tokens.ts:58]: `[^\w.]([\w-]+)\.([\w-]+)`

File: modules/test/poshi/poshi-vscode/src/lib/tokens.ts [hunk 4]
```diff
@@ -20,7 +20,7 @@ const tokenPatternMap = {
 	/**
 	 * matches "test TestCaseName {"
 	 */
-	testCaseName: /test ([A-Z][A-Za-z]+) \{/g,
+	testCaseName: /test ([\w-]+) \{/g,
 
 	/**
 	 * matches "${vari|able}"
```
Comment (kenjiheigel) [inline modules/test/poshi/poshi-vscode/src/lib/tokens.ts:26]: Variables are also `[\w-]+`

**Comment by drewbrokke** [comment]
See https://github.com/drewbrokke/liferay-portal/pull/1463

**Pull request closed without merging**